lib/types.ts

```diff
@@ -250,7 +250,7 @@ export type DayRange = (typeof CHART_RANGES)[number]
 export type SoftwareDetailItem = {
   id: string
   label: React.ReactNode
-  className: string
+  className?: string
```

Comment:
Most don't need classNames anymore since we are laying them all out with grid areas now, which is using the `style` prop instead (not natively supported with tailwind)